usr/share/rear/layout/save/GNU/Linux/280_multipath_layout.sh

```diff
@@ -23,7 +23,11 @@ while read dm_name junk ; do
         slaves="$slaves$(get_device_name ${slave##*/}),"
     done
 
-    echo "multipath /dev/mapper/$dm_name $dm_size ${slaves%,}" >> $DISKLAYOUT_FILE
+    dm_disktype=$(parted -s $dev_name print | grep -E "Partition Table|Disk label" | cut -d ":" -f "2" | tr -d " ")
+
+    echo "# Multipath /dev/mapper/$dm_name"
+    echo "# Format: multipath <devname> <size(bytes)> <partition label type> <slaves>"
```

Comment:
Optionally while you are at it:

Could you also document the multipath syntax
in the "Disk layout file syntax" section in doc/user-guide/06-layout-configuration.adoc
https://github.com/rear/rear/blob/master/doc/user-guide/06-layout-configuration.adoc

Reply:
Added in the updated patch.